### PR TITLE
[T4A6][F10-B1] Calvin Yang JiaWei

### PR DESCRIPTION
**Pull request by origiri**

### CODE REVIEW

**Comment by origiri** [comment]
Ready for review

File: src/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+public class UtilsTest {
+    
+    /**
+     * Test isAnyNull()
+     */
+    
+    @Test
+    public void isAnyNull_noArgs() {
+        assertFalse(Utils.isAnyNull());
```
Comment (Candiie) [inline src/seedu/addressbook/common/UtilsTest.java:17]: :+1:  for using assertFalse.

File: src/seedu/addressbook/common/UtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    
+    @Test 
+    public void isAnyNull_manyNoNullObject() {
+        assertFalse(Utils.isAnyNull("notNull", "notNull"));
+    }
+    
+    @Test
+    public void isAnyNull_oneNullObject() {
+        Object nullObject = null;
+        assertTrue(Utils.isAnyNull(nullObject));
```
Comment (Candiie) [inline src/seedu/addressbook/common/UtilsTest.java:33]: :+1:  for using assertTrue.

**Comment by Candiie** [comment]
Some comments added. Please close the PR after reading comments.

Hi Hi Calvin, try to split your commits by the type of tests implemented. Currently it seems alright to commit them together because they are currently short and somewhat simple. However in the future, do take note to split them - one for every significant changes or split commits if they are not exactly similar even though they might be related. 

Otherwise, if you find it too troublesome to explicitly split everything, try to at least group them by similarity and commit them separately by groups, for example in your case, e.g. "Testing for elementsAreUnique()....." or "Asserting for true cases for elements ....." 

~ Chuan Wei Candiie

**Pull request closed without merging**